src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -82,11 +91,53 @@ object WdlExpression {
     val parseTree = parser.parse_e(tokens, new WdlSyntaxErrorFormatter(terminalMap))
     new WdlExpression(parseTree.toAst)
   }
+
+  def toString(ast: AstNode, highlighter: SyntaxHighlighter = NullSyntaxHighlighter): String = {
```

Comment:
Sure, that's another way to do it.  I've thought a lot about it and I think a vast majority of cases have 1 highlighter per VM.  I tried to contrive a case where that'd not be the case, and the best I can come up with is the web server might have multiple was of exporting a WDL file.  This would mean there'd be at least 2 highlighters (e.g. LaTeX and HTML) used within one JVM.

When I wrote it, I didn't feel like it was to obtrusive to be like this... Maybe if it becomes a bigger problem we can switch implementations?